.github/workflows/continuous-integration.yml

```diff
@@ -274,7 +274,7 @@ jobs:
 
       - name: Verify apidoc changes
         run: |
-          apidoc_file_path='docs/src/api_reference.md'
+          apidoc_file_path='docs/src/cloudnative-pg.v1.md'
```

Comment:
why changing the name of the file here? I mean because it's the api reference cloudnative-pg sound like an index to make, this is the only changes that rings a bell on me